docs/package.json

```
@@ -0,0 +1,23 @@
+{
+  "name": "@xstate/docs",
```

Given our packages use the `@xstate` scope and this is not a package per se we might want to avoid using this scope like that to avoid confusion. It's a really minor thing though so if you don't agree - just disregard the comment.

```suggestion
  "name": "xstate-docs",
```